testgrid/cmd/updater/main.go

```diff
@@ -121,17 +123,24 @@ type Started struct {
 	Repos       map[string]string `json:"repos"` // {repo: branch_or_pull} map
 }
 
+// Finished holds the finished.json values of the build
 type Finished struct {
-	Timestamp  int64    `json:"timestamp"` // epoch seconds
+	// Timestamp is epoch seconds
+	Timestamp  int64    `json:"timestamp"`
 	Passed     bool     `json:"passed"`
 	JobVersion string   `json:"job-version"`
 	Metadata   Metadata `json:"metadata"`
 	running    bool
 }
 
-// infra-commit, repos, repo, repo-commit, others
+// Metadata holds the finished.json values in the metadata key.
+//
+// Metadata values can either be string or string map of strings
+//
+// Special values: infra-commit, repos, repo, repo-commit, others
```

Comment:
Is there documentation to link to for these values?

Reply:
Not externally, and not sure we want to keep any of this

Reply:
Fair enough.